src/app/shared/interfaces/coefficient.ts

```diff
@@ -0,0 +1,6 @@
+export interface Coefficient {
+    role: number;
+    coefficient: number;
+    hours_option: number;
+    workload_option: number;
```

Comment:
Mark all fields are optional, and add other fields existed in the definition of the ORM model, such as `campus_event`, `off_campus_event`.